mangaki/mangaki/templates/mangaki/reco_list.html

```diff
@@ -44,32 +28,25 @@
 <br />
 
     <ul class="nav nav-tabs">
-        <li role="presentation"{% if category == 'all' %} class="active"{% endif %}><a href="{% url 'reco' %}">Tout</a></li>
-        <li role="presentation"{% if category == 'anime' %} class="active"{% endif %}><a href="{% url 'reco' %}?category=anime"><em>Anime</em> seulement</a></li>
-        <li role="presentation"{% if category == 'manga' %} class="active"{% endif %}><a href="{% url 'reco' %}?category=manga">Manga seulement</a></li>
+        <li role="presentation"{% if category == 'all' %} class="active"{% endif %}><a href="{% url 'reco' %}?algo={{ algo }}">Tout</a></li>
+        <li role="presentation"{% if category == 'anime' %} class="active"{% endif %}><a href="{% url 'reco' %}?algo={{ algo }}&category=anime"><em>Anime</em> seulement</a></li>
+        <li role="presentation"{% if category == 'manga' %} class="active"{% endif %}><a href="{% url 'reco' %}?algo={{ algo }}&category=manga">Manga seulement</a></li>
     </ul>
     <br />
 <div class="alert alert-warning" role="alert" id="sorry" style="display: none"></div>
 {% include "works.html" with works=reco_list is_dpp=False %}
 <div class="row margin">
     <div class="col-xs-12">
-        <a class="btn btn-primary btn-lg" href="{% url 'reco' %}?category={{ category }}&editor={{ editor }}" style="display: block; margin: auto">Je veux d'autres recommandations&nbsp;! <br class="visible-xs">(Après avoir noté celles-ci.)</a>
+        <a class="btn btn-primary btn-lg" href="{% url 'reco' %}?category={{ category }}&algo={{ algo }}" style="display: block; margin: auto">Je veux d'autres recommandations&nbsp;! <br class="visible-xs">(Après avoir noté celles-ci.)</a>
```

Comment:
Instead of putting the parameters in the URL? I think it's because those are parameters, not different pages.